main/src/ca/uwaterloo/flix/language/phase/Verifier.scala

```diff
@@ -45,6 +45,15 @@ object Verifier {
     checkEq(decl.tpe, ret, decl.loc)
   }
 
+  private def checkStructType(tpe: MonoType, sym0: Symbol.StructSym, loc: SourceLocation): Unit = {
```

Comment:
Add a comment and move it down with all the other check functions near the bottom of the file :)